From ac38373715248dea69eb71d0dad21f9672084f6c Mon Sep 17 00:00:00 2001
From: Ken CJ Chou <kenchou0731@gmail.com>
Date: Wed, 3 Jun 2020 09:26:05 +0000
Subject: [PATCH 3/3] Define CONFIG_FILE "/etc/fw_env.config"

---
 include/fw_env.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/fw_env.h b/include/fw_env.h
index 78c803c..0eea99a 100644
--- a/include/fw_env.h
+++ b/include/fw_env.h
@@ -13,6 +13,8 @@
  */
 #define FW_ENV_API_VERSION	1
 
+#define CONFIG_FILE     "/etc/fw_env.config"
+
 struct env_opts {
 #ifdef CONFIG_FILE
 	char *config_file;
-- 
2.20.1

